src/yogita/Assignment_6/PropertyFile.java

```diff
@@ -0,0 +1,101 @@
+package yogita.Assignment_6;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.List;
+import java.util.Properties;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.Select;
+
+public class PropertyFile {
+	static WebDriver driver;
+
+	static void launchBrowser() {
+		String path = "./resources/windows/chromedriver.exe";
+		System.setProperty("webdriver.chrome.driver", path);
+		driver = new ChromeDriver();
+		System.out.println("STEP-Go To WebSite");
+		driver.get("file:///E:/java%20program/SeleniumTechnoOct2020/resources/forms/First_Form_4.html");
+		System.out.println("Website open successfully.");
+		driver.manage().window().maximize();
+	}
+
+	static void propertyFileDemo() throws IOException {
+		File file = new File("./src/yogita/config1/configData.properties");
```

Comment:
suggestion : Create separate function for the reading properties file.

Reply:
Ok